### DISCUSSION TITLE
[SOLVED] fonts and font sizes

### DISCUSSION THREAD

**Discussion by Skywriter960** [martinrotter/rssguard#633] [Q&A]
Hello, 

now I ask where I can temporarily remove individual feeds from the collective retrieval and where I can set the fonts and font sizes. With the integrated browser, the font is partly so small that I can decipher them "only with nose on the display" despite display scaling at 125% - despite glasses ... 😉

Or have I overlooked something when familiarizing myself with the program?

Greetings,
Skywriter

This post was posted with a translation program.

**Comment by martinrotter** [comment]
Hi, you wrote me that german email?

As for switching individual feeds "off" - this was recently added and will be published with upcoming RSS Guard 4.2.0.

You can zoom in/out internal viewer with `CTRL+mousewheel`. You can change fonts in `Tools -> Settings -> Articles`.

**Comment by Skywriter960** [comment]
Hello and good day,

Yes, I had also sent an email.

Thanks for the quick reply!

Oops, then I must have been 'something' too fast when configuring ... With the mouse wheel is a good tip!

Nice that my other wish is already in the pipeline.

This program I use, because there is fortunately also a portable version for this. At home I have only a snail connection and have to go to a public PC with proxy ...

Thanks a lot and have a nice day,
Skywriter


Translated with www.DeepL.com/Translator (free version)
